lms/static/js/learner_dashboard/views/unenroll_view.js

```diff
@@ -0,0 +1,117 @@
+(function(define) {
+    'use strict';
+    define(['backbone',
+        'jquery',
+        'underscore',
+        'gettext',
+        'edx-ui-toolkit/js/utils/html-utils'
+    ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             HtmlUtils
+         ) {
+             return Backbone.View.extend({
+                 el: '.unenroll-modal',
+
+                 initialize: function(options) {
+                     this.urls = options.urls;
+
+                     $('.action-unenroll').click(function(event) {
+                         var $element = $(event.target),
+                             trackInfo = $element.data('track-info'),
+                             courseNumber = $element.data('course-number'),
+                             courseName = $element.data('course-name'),
+                             certNameLong = $element.data('cert-name-long'),
+                             survey, i;
+                         HtmlUtils.setHtml(
+                             $('#track-info'), HtmlUtils.HTML(window.interpolate(trackInfo, {
+                                 course_number: HtmlUtils.joinHtml(
+                                    HtmlUtils.HTML("<span id='unenroll_course_number'>"),
+                                    courseNumber,
+                                    HtmlUtils.HTML('</span>')
+                                 ),
+                                 course_name: HtmlUtils.joinHtml(
+                                    HtmlUtils.HTML("<span id='unenroll_course_name'>"),
+                                    courseName,
+                                    HtmlUtils.HTML('</span>')
+                                 ),
+                                 cert_name_long: HtmlUtils.joinHtml(
+                                    HtmlUtils.HTML("<span id='unenroll_cert_name'>"),
+                                    certNameLong,
+                                    HtmlUtils.HTML('</span>')
+                                 )
+                             }, true))
+                         );
+                         HtmlUtils.setHtml(
+                             $('#refund-info'), HtmlUtils.HTML($element.data('refund-info'))
+                         );
+                         $('#unenroll_course_id').val($element.data('course-id'));
+
+                         // Randomize survey option order
+                         survey = document.querySelector('.options');
+                         for (i = survey.children.length; i >= 0; i--) {
+                             survey.appendChild(survey.children[Math.random() * i | 0]);
+                         }
+                     });
```

Comment:
It would be great if you could move the anonymous functions called when these event handlers are fired into their own, named, functions to make the code easier to read.